README.md

```diff
@@ -314,6 +316,51 @@ Name            | Effect |
 `vagrant-box`    | Build the Vagrant box image using `packer` |
 `vagrant-prep`    | Shortcut for building everything needed for `make run` |
 
+# Deployment Customizations
+
+## Customize The Application Domain
+
+In a standard installation the domain used by applications pushed to
+CF is the same as the domain configured for CF itself.
```

Comment:
No need for `itself` at the end of the phrase.

Reply:
ok

Reply:
Done with commit 05aaa3c57e9da760304a8f1cd5e48362695b77d0